maquetado/tech.scss

```diff
@@ -0,0 +1,43 @@
+.icon {
+  height: $tech-logo-size;
+}
+
+.title-sub {
+  font-size: $text-medium;
+  margin: 10px 0;
+}
+
+.title-relase {
+  font-size: $text-medium;
+  font-weight: bold;
+  margin: 10px 0;
+}
+
+.title-description {
+  font-size: $text-medium;
+  margin: 10px 0;
+}
+// cuadro verde
+.tech {
+  align-items: center;
+  background-color: $green;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  margin: 10px 4px;
+  position: relative;
+  text-align: center;
+  width: 200px;
```

Comment:
Add a break line. Check this [style guide](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#indentation)